index.html

```diff
@@ -221,7 +225,7 @@ <h5>
           <div class="horizontal-container">
             <img class="item" src="/images/cardImage.jpg" />
             <div class="text-container">
-              <h5>New Arriavles</h5>
+              <h5>New Arrivales</h5>
```

Comment:
I think you've made a typo here, this might be "Arrivals"

Reply:
> I think you've made a typo here, this might be "Arrivals"

Thank you I've fixed it.